llvm/test/CodeGen/AArch64/hadd-combine.ll

```diff
@@ -859,6 +859,21 @@ define <4 x i32> @urhadd_v4i32(<4 x i32> %x) {
   ret <4 x i32> %r
 }
 
+define <4 x i32> @fixedwidth(<4 x i32> %a0, <4 x i32> %a1)  {
```

Comment:
Maybe rename @uhadd_fixedwidth_v4i32 ? You will be adding more fixedwidth tests in this file

Reply:
Yes, sure.